Makefile

```diff
@@ -678,7 +678,9 @@ ifdef ASSERT_STATUS_CHECKED
 		compaction_job_test \
 		compaction_job_stats_test \
 	        io_tracer_test \
-		merge_helper_test \
+		io_tracer_parser_test \
+		prefetch_test \
+	        merge_helper_test \
```

Comment:
nit: change the spaces to tabs

Reply:
https://giphy.com/gifs/producthunt-silicon-valley-spaces-tabs-l0IylSajlbPRFxH8Y